apps/rush-lib/src/cli/actions/CustomRushAction.ts

```diff
@@ -148,6 +155,13 @@ export class CustomRushAction extends BaseRushAction {
       parameterShortName: '-v',
       description: 'Display the logs during the build, rather than just displaying the build status summary'
     });
+    if (this.options.actionVerb === 'build') {
+      this._ignoreDownstream = this.defineFlagParameter({
```

Comment:
In some places you're saying "ignoreDownstream" and in others you say "invalidateDownstream." Can you use consistent verbage?

Reply:
Sure

---
In reply to: [164220753](https://github.com/Microsoft/web-build-tools/pull/514#discussion_r164220753) [](ancestors = 164220753)